EIPS/eip-4844.md

```diff
@@ -468,10 +470,11 @@ instead, they go into the `BeaconBlockBody`. This means that there is now a part
 Blob transactions have a large data size at the mempool layer, which poses a mempool DoS risk,
 though not an unprecedented one as this also applies to transactions with large amounts of calldata.
 
-We recommend two solutions:
+By only broadcasting announcements for blob transactions, nodes have control over which and how many transactions to receive,
+allowing them to throttle throughput to an acceptable level.
+[EIP-5793](./eip-5793.md) will give further fine-grained control to nodes by extending the `NewPooledTransactionHashes` announcement messages to include the transaction type and size.
 
-- include a 1.1x (or potentially higher) data gasprice bump requirement to the mempool replacement rules
-- modify the Ethereum Wire Protocol to stop automatically broadcasting large transactions
+In addition, we recommend including a 1.1x data gasprice bump requirement to the mempool transaction replacement rules.
```

Comment:
```suggestion
In addition, we recommend including a 1.1x data gasprice bump requirement to the mempool transaction replacement rules.
```

Is the "recommendation" added here as a specification? Could you clarify it's a comment or it's a specification?